server/container_create.go

```diff
@@ -245,7 +153,7 @@ func addImageVolumes(rootfs string, s *Server, containerInfo *storage.ContainerI
 			}
 			// Label the source with the sandbox selinux mount label
 			if mountLabel != "" {
-				if err1 := label.Relabel(src, mountLabel, true); err1 != nil && err1 != unix.ENOTSUP {
+				if err1 := label.Relabel(src, mountLabel, true); err1 != nil && err1 != syscall.ENOTSUP {
```

Comment:
Instead of backsliding to syscall, can we just move this SELinux code into a `*_linux.go` helper function?  That way we can skip the SELinux stuff on Darwin, instead of compiling and then crashing at runtime.

Reply:
Yes just make an fixSelinuxLabel function and hide this in there.

Reply:
done. will push shortly